package.json

```diff
@@ -34,28 +37,31 @@
     "@storybook/react": "3.4.3",
     "autoprefixer": "8.4.1",
     "babel-core": "6.26.3",
-    "babel-eslint": "9.0.0",
+    "babel-eslint": "10.0.0",
     "babel-loader": "7.1.4",
     "babel-plugin-syntax-class-properties": "6.13.0",
     "babel-plugin-syntax-object-rest-spread": "6.13.0",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-preset-env": "1.7.0",
-    "babel-preset-react": "^6.11.1",
+    "babel-preset-react": "6.24.1",
     "cross-env": "5.1.5",
     "css-loader": "0.28.11",
-    "eslint": "5.5.0",
-    "eslint-config-airbnb": "^10.0.1",
-    "eslint-plugin-import": "^1.13.0",
-    "eslint-plugin-jsx-a11y": "^2.1.0",
-    "eslint-plugin-react": "^6.1.1",
+    "eslint": "5.6.0",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-jsx-a11y": "6.1.1",
+    "eslint-plugin-prettier": "^3.0.0",
+    "eslint-plugin-react": "7.11.1",
     "html-webpack-plugin": "3.2.0",
     "husky": "0.14.3",
     "jest": "22.4.3",
     "lint-staged": "7.1.0",
     "mini-css-extract-plugin": "^0.4.0",
     "postcss-assets": "5.0.0",
     "postcss-loader": "2.1.5",
+    "prettier": "1.14.3",
+    "prop-types": "15.6.2",
```

Comment:
Why `prop-types` is added as a dev dependency? It is used as a production dependency.